extension/src/project/addFile2Project.ts

```diff
@@ -90,3 +90,17 @@ async function SelectLspFiles() {
     if (fileUris && fileUris.length > 0)
         return Promise.resolve(fileUris);
 }
+
+
+export async function SelectFolder(): Promise<vscode.Uri[]> {
+    let label = AutoLispExt.localize("autolispext.project.addfolder.openlabel", "Add Folder");
```

Comment:
Thanks for your patience!

Sorry I didn't make me understood on this item. 
Please see the GIF: 
https://github.com/Autodesk-AutoCAD/AutoLispExt/pull/51#discussion_r528318730

And I mean that we only improve the "add file to project" menu item and make it support to select a folder.